packages/terra-form-textarea/tests/jest/Textarea.test.jsx

```diff
@@ -122,29 +123,29 @@ it('should not set the textarea to resizable when viewed on browser with msMaxTo
   spyOn(window, 'matchMedia').and.returnValue({ matches: true });
   navigator.msMaxTouchPoints = 1;
 
-  const textarea = <Textarea isAutoResizable />;
+  const textarea = <Textarea ariaLabel="label" isAutoResizable />;
   const wrapper = render(textarea);
   expect(wrapper).toMatchSnapshot();
 
   navigator.msMaxTouchPoints = null;
 });
 
 it('should set the textarea to disabled when passed into the component', () => {
-  const textarea = <Textarea disabled />;
+  const textarea = <Textarea ariaLabel="label" disabled />;
   const wrapper = render(textarea);
   expect(wrapper).toMatchSnapshot();
 });
 
 it('should set the textarea to invalid when isInvalid is passed into the component', () => {
-  const textarea = <Textarea isInvalid />;
+  const textarea = <Textarea ariaLabel="label" isInvalid />;
   const wrapper = render(textarea);
   expect(wrapper).toMatchSnapshot();
 });
 
 it('should set the ref when refCallback is passed into the component', () => {
   const refCallback = jest.fn();
 
-  const textarea = <Textarea refCallback={refCallback} />;
+  const textarea = <Textarea ariaLabel="label" refCallback={refCallback} />;
   const wrapper = mount(textarea);
 
```

Comment:
No need to update all the tests with this new prop i guess . as this is not the required one here.

Reply:
Fixed here 0aa2ae3.